content/blog/understanding-hpe-greenlake-identity-access-management.md

```diff
@@ -103,47 +102,40 @@ The invited user receives an email inviting them to join the tenant. Once the us
 ## Creating and modifying User Groups
 
 1. Access the User Management Service within the **wrench** icon.
-
 2. Select the **User Groups** tab
-
 3. To modify an existing user group, click on it from the list of user groups
+
    * Members can be added by selecting the **Members** tab and selecting *Add Members* under the **Actions** pull-down
    * Members can be removed from the user group by clicking on the **Trash** icon beside the member name
-
 4. To create a new user group, from the **User Groups** tab, click on the *Create User Group* button
-
 5. Enter a user group name and description
-
 6. Once the user group has been created it may be modified using the instructions above
 
 ## Creating and Modifying Spaces
 
 1. Access the Management Service within the **wrench** icon.
-
 2. Select the **Spaces** tab
-
 3. To modify an existing space, click on it from the list of spaces
 
    1. Users and user groups can be added by selecting the **Assignments** tab and selecting *Create Assignment* under the **Actions** pull-down
-
    2. Select appropriate role(s) for the space
+
       * Users and user groups can be removed from the space by clicking on the **Trash** icon beside the subject name
       * Resources can be added and removed from the space by selecting the **Resources** tab and selecting *Update Resources* under the **Actions** pull-down
-
 4. To create a new space, click on the **Create Space** button
+
    * Enter a space name and parent space   
    * Select **Resources** by expanding the **All Resources** list   
 
-
 # Let's Explore Some Fictitious Customer Examples
 
 ## Example 1: ACME Corp.
 
 ### Design
 
-![](/img/iam-document-example-1-tenant-1-1-.jpg "Example 1: ACME Corp.")
+![](/img/iam-document-example-1-tenant-1-1a.jpg "Example 1: ACME Corp.")
 
-Customer ACME Corp. has users spread around the globe. The users' email addresses all take the form of [xyz@acmecorp.com.](mailto:xyz@acmecorp.com.) ACME Corp. has several departments that are also distributed around the globe. ACME Corp. wishes to use HPE GreenLake Private Cloud to create and manage virtual machines and containers on behalf of the various departments. They also wish to use HPE GreenLake for ML Ops to examine their internal data and perform AI operations upon it and use HPE GreenLake for containers to run a sales application.
+Customer ACME Corp. has users spread around the globe. The users' email addresses all take the form of [xyz@acmecorp.com.](mailto:xyz@acmecorp.com.) ACME Corp. has several departments that are also distributed around the globe. ACME Corp. wishes to use HPE GreenLake Private Cloud Enterprise to create and manage virtual machines and containers on behalf of the various departments. They also wish to use HPE GreenLake for ML Ops to examine their internal data and perform AI operations upon it and use HPE GreenLake for containers to run a sales application.
```

Comment:
This should be ...to use HPE GreenLake for Private Cloud Enterprise to create....

(the "for" is missing).